dagshub/common/logging.py

```diff
@@ -24,13 +24,8 @@ def format(self, record):
 
 
 # create logger with 'spam_application'
-logger = logging.getLogger("DagsHub")
-logger.setLevel(logging.DEBUG)
-
+logger = logging.getLogger()
```

Comment:
Code libraries (imo) should never ever fiddle with the root logger, that's gonna ruin some user's day.
Due to how python's logging inheritance works, we can actually do this:
```python
logger = logging.getLogger("dagshub")
```

And as long as every other class in the library gets a logger via 
```python
logger = logging.getLogger(__name__) # say dagshub.streaming.filesystem
```
the logging calls will bubble up to this logger and we should get this format for the whole library.

Reply:
I'll just not temper with this at all, WDYT?

Reply:
Good idea, let's just leave formatting for the CLI, and in all other usecases just defer to however the user set up their logging